aries-site/src/examples/components/pagination/PaginationServerTableExample.js

```diff
@@ -104,36 +116,51 @@ export const PaginationServerTableExample = () => {
       </Heading>
 
       <Box>
-        <DataTable
-          aria-describedby="server-side-pagination-heading"
-          columns={columns}
-          data={data}
-          sort={{ ...sort, external: true }}
-          step={limit}
-          onSort={opts => setSort(opts)}
-          fill
-          gap="small"
-        />
+        <Box overflow="auto">
+          <DataTable
+            aria-describedby="server-side-pagination-heading"
+            columns={columns}
+            data={data}
+            sort={{ ...sort, external: true }}
+            onSort={opts => setSort(opts)}
+            fill
+            gap="small"
+          />
+        </Box>
 
         {numberItems > limit && (
           <Box
             direction="row-responsive"
-            fill="horizontal"
             border="top"
-            justify="end"
-            pad={{ vertical: 'xsmall' }}
+            gap="large"
+            pad={{ vertical: 'small' }}
           >
-            <Text>
-              Showing {pageResultStart}-{pageResultEnd} of {numberItems}
-            </Text>
-            <Pagination
-              step={limit}
-              numberItems={numberItems}
-              page={page}
-              onChange={opts => setPage(opts.page)}
-              direction="row"
-              flex={false}
-            />
+            {!smallScreenSize && (
```

Comment:
I'm wondering if the ordering of this should be altered so that the results summary ("Showing X of Y") is on the far left, then we have the controls that affect the view grouped. So something like:

```
<Box direction="row" align="center" gap="small">
      <Text>Showing X of Y</Text>
      <Box direction ="row" gap="small" align="center">
            <ResultsPerPageSelect />
            <Pagination />
      </Box>
</Box>